server/router/router.go

```diff
@@ -0,0 +1,27 @@
+package router
+
+import (
+	"app/controllers"
+
+	"github.com/gin-gonic/gin"
+)
+
+func InitRouter() {
+
+	r := gin.Default()
+
+	router := r.Group("/v1")
+	{
+		// users
+		user_router := router.Group("/user")
+		{
+			user_router.POST("/", controllers.InsertUser)
+			user_router.GET("/", controllers.GetUsers)
+			user_router.GET("/:id", controllers.GetUser)
+			user_router.PATCH("/", controllers.UpdateUser)
+			user_router.DELETE("/", controllers.DeleteUser)
```

Comment:
Could we also patch and delete by id, or is that not recommended?

Reply:
@Julius-R  I actually forgot about that. It should be based on the Id

Reply:
@Julius-R PATCH and DELETE are now update to query by id